app/__init__.py

```diff
@@ -3,7 +3,7 @@
 from flask_jwt_extended import JWTManager
 from instance.config import config
 
-from app.api.v1.views.incident import RedFlagRecords, RedFlagRecord
+from app.api.v1.views.incident import (RedFlagRecords, RedFlagRecord, RedFlagRecordLocation, RedFlagRecordComment)
```

Comment:
This line is longer than recommended. Trim it to 79 characters. You can push some imports to the next line.
from ... import ( a, b
                           c, d)